packages/containers-shared/tests/docker-context.test.ts

```diff
@@ -0,0 +1,65 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+import { resolveDockerHost } from "../src/utils";
+
+const mockedDockerContextLsOutput = `{"Current":true,"Description":"Current DOCKER_HOST based configuration","DockerEndpoint":"unix:///current/run/docker.sock","Error":"","Name":"default"}
+{"Current":false,"Description":"Docker Desktop","DockerEndpoint":"unix:///other/run/docker.sock","Error":"","Name":"desktop-linux"}`;
+
+vi.mock("node:child_process");
+
+describe.skipIf(process.platform !== "linux" && process.env.CI === "true")(
```

Comment:
`process.platform !== "linux" && process.env.CI === "true"`

If think I have already seen this condition but I can't remember why it is.

Would factoring the code into an helper function with some comment about the rationale help understanding/maintaining this code?